README.md

```diff
@@ -11,4 +11,10 @@ A collection of [NestJS](https://docs.nestjs.com) components.
 
 A simple interceptor formatting a HTTP response with a `Link` header and a `Content-Range`.
 
-See [the documentation here](./packages/nestjs-pagination/README.md).
\ No newline at end of file
+See [the documentation here](./packages/nestjs-pagination/README.md).
+
+## NestJS Logging interceptor
+
+A simple NestJS interceptor catching request details and loggin it using the built-in [Logger](https://docs.nestjs.com/techniques/logger#logger) class. It will use the default Logger implementation unless you pass your own to your Nest application.
```

Comment:
```suggestion
A simple NestJS interceptor catching request details and logging it using the built-in [Logger](https://docs.nestjs.com/techniques/logger#logger) class. It will use the default Logger implementation unless you pass your own to your Nest application.
```